src/js/utils/filters/ValueFilter.ts

```diff
@@ -0,0 +1,40 @@
+import { FilterInterface } from 'utils/filters/FilterInterface';
+import { ValueFilterInterface } from 'utils/filters/ValueFilterInterface';
+
+export class ValueFilter implements FilterInterface, ValueFilterInterface {
+    filterKey: string = '';
+    filterValues: string[] = [];
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    items: any[] = [];
```

Comment:
is it possible to create a generic for filterkey and enforce that it is a property string[] in items?